parachains/runtimes/assets/statemint/src/assets_api.rs

```diff
@@ -0,0 +1,34 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2018-2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later WITH Classpath-exception-2.0
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! Runtime API definition for assets.
+
+use codec::Codec;
+use sp_std::vec::Vec;
+
+sp_api::decl_runtime_apis! {
+	pub trait AssetsApi<AccountId, AssetBalance, AssetId>
```

Comment:
@joepetrowski [here](https://github.com/paritytech/cumulus/pull/2133#issue-1554990311) you mentioned `assets-common`, so I think this `AssetsApi` is also very hot candidate for that, can we start with `assets-common` here? I guess, this PR will go to the master before our ForeignAssets stuff

Reply:
Yeah I'm open to that, don't have time to refactor it myself in the near-term, and I would like to get 2133 in before the next release. But if someone can work on that and this fits in then by all means go for it.